README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
   <br>
-  <img alt="OpenSauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
+  <img alt="OpenSauced" src="https://github.com/open-sauced/assets/blob/main/logos/logo-on-dark.png" width="300px">
```

Comment:
Seeing this live. The image seems small. can you swap it with this one
```suggestion
  <img alt="OpenSauced" src="https://github.com/open-sauced/assets/blob/main/logo.png?raw=true" width="300px">
```
![Screenshot 2024-04-25 at 11 43 58 AM](https://github.com/open-sauced/app/assets/5713670/99367bac-4ad5-4f70-a544-848ab8130604)

Reply:
oh nvm, I see the width is 300px. can we remove that to make it bigger?

Reply:
I will remove the width of 300px, @bdougie. Can you check if the picture looks good now ?